x-pack/plugins/siem/server/lib/detection_engine/signals/signal_params_schema.ts

```diff
@@ -39,5 +39,7 @@ export const signalParamsSchema = () =>
     type: schema.string(),
     references: schema.arrayOf(schema.string(), { defaultValue: [] }),
     version: schema.number({ defaultValue: 1 }),
+    // For backwards compatibility with customers that had a data bug in 7.7. Once we use a migration script please remove this.
+    lists: schema.maybe(schema.arrayOf(schema.object({}, { unknowns: 'allow' }))),
```

Comment:
NOTE: Ensure that this is exactly the same as here from 7.7 to be tolerant of the bug.

https://github.com/elastic/kibana/blob/7.7/x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_params_schema.ts#L42